fiftyone/plugins/skills.py

```
@@ -0,0 +1,253 @@
+            not skill.name
+            or os.path.basename(skill.name) != skill.name
+            or skill.name in (".", "..")
+        ):
+            raise ValueError(f"Invalid skill name: {skill.name!r}")
+
+        skill_central = os.path.join(central_dir, skill.name)
+        os.makedirs(skill_central, exist_ok=True)
+
+        dest = os.path.join(skill_central, os.path.basename(skill.path))
+        shutil.copy2(skill.path, dest)
+        logger.info(f"Copied skill '{skill.name}' to '{skill_central}'")
```

Ruff (0.15.7)

[warning] Logging statement uses f-string

(G004)